packages/client/test/FakeBrubeckNode.ts

```diff
@@ -0,0 +1,81 @@
+import debug from 'debug'
+import { pull } from 'lodash'
+import { EthereumAddress, StreamMessage, StreamPartID } from 'streamr-client-protocol'
+import { DestroySignal } from '../src/DestroySignal'
+import { FakeBrubeckNodeRegistry } from './FakeBrubeckNodeRegistry'
+
+const log = debug('Streamr:FakeBrubeckNode')
+
+export class FakeBrubeckNode {
+
+    private address: EthereumAddress
+    public subsribedStreamParts: Set<StreamPartID> = new Set()
+    public messageListeners: ((msg: StreamMessage) => void)[] = []
+    private fakeBrubeckNodeRegistry: FakeBrubeckNodeRegistry
+
+    constructor(
+        address: EthereumAddress,
+        fakeBrubeckNodeRegistry: FakeBrubeckNodeRegistry,
+        destroySignal: DestroySignal | undefined,
+        name?: string
+    ) {
+        this.address = address.toLowerCase()
+        this.fakeBrubeckNodeRegistry = fakeBrubeckNodeRegistry
+        if (destroySignal !== undefined) {
+            destroySignal.onDestroy(() => this.destroy())
+        }
+        log(`Created${name ? ' ' + name : ''}: ${address}`)
+    }
+
+    getAddress() {
+        return this.address
+    }
+
+    // eslint-disable-next-line class-methods-use-this
+    startNode() {
+    }
```

Comment:
If I remember correctly this should delegate to `getNode`.

Reply:
or `throw new Error('not implemented')`

Reply:
It's a no-op in fake environment (the real BrubeckNode startNode now returns Promise<void>)